src/test/java/uk/gov/register/presentation/functional/testSupport/DBSupport.java

```diff
@@ -94,24 +97,19 @@ private static void publishMessages(Connection connection, String registerName,
         publishMessages(connection, registerName, messagesWithSerialNumbers);
     }
 
-    private static Optional<Long> publishMessages(Connection connection, String registerName, SortedMap<Integer, String> messages) throws SQLException {
-        Optional<Long> lastUpdatedTime = Optional.empty();
+    private static void publishMessages(Connection connection, String registerName, SortedMap<Integer, String> messages) throws SQLException {
         for (SortedMap.Entry<Integer, String> entry : messages.entrySet()) {
             int serialNumber = entry.getKey();
             String message = entry.getValue();
             try (PreparedStatement insertPreparedStatement = connection.prepareStatement("Insert into ordered_entry_index(serial_number,entry,leaf_input) values(?,?, ?)")) {
-                Long updateTime;
                 insertPreparedStatement.setObject(1, serialNumber);
                 insertPreparedStatement.setObject(2, jsonbObject(message));
-                insertPreparedStatement.setString(3, CTLeafInputGenerator.createLeafInputFrom(itemData(message), updateTime = System.currentTimeMillis()));
+                insertPreparedStatement.setString(3, CTLeafInputGenerator.createLeafInputFrom(itemData(message), System.currentTimeMillis()));
```

Comment:
It was picking timestamp of last entry added into the table which was used to assert the register's last_updated time in tests.